src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -103,9 +105,14 @@ object PushdownUtilities {
       case NoPushdown() => false
     }
 
-  def assetIdsFromWrappedArray(wrappedArray: String): Seq[Long] =
+  def idsFromWrappedArray(wrappedArray: String): Seq[Long] =
     wrappedArray.split("\\D+").filter(_.nonEmpty).map(_.toLong)
 
+  def stringSeqFromWrappedArray(wrappedArray: String): Seq[String] = {
+    val regQuotes: Regex = "\"(.*?)\"|'(.*?)'".r
+    regQuotes.findAllIn(wrappedArray).toSeq
+  }
+
```

Comment:
@hakontro I think this should work for extracting externalIds from a WrappedArray of Strings. It grabs substrings between quotation or double quotations.